odata-csdl/Appendix.md

```diff
@@ -152,6 +152,254 @@ https://www.ogc.org/standard/sfa/.
 $$$isec$$$
 :::
 
+-------
+
+# Appendix ##asec Model elements, properties and relationships
+
+## ##subasec Overview
+
+This appendix describes the model elements, their properties and relationships of the abstract OData model. The XSD schema is the normative document for the model elements and their properties. Yet, because of the limitations of XSD schema, it only contains the parent-child relationships of the model elements. For example the type of a property is represented as a string in XML and JSON, a string of the qualified name of the type. In the abstract model, it is a reference to the element that is the type.
```

Comment:
Actually _the_ normative document is the CSDL specification, the XSD only describes necessary conditions and is not sufficient.

Reply:
That's sort of my point and in essence my motivation for this document. 
The specification is not suitable to be machine readable (one might say a meta model). This proposal is a much more structured document that states how properties are to be interpreted (values, references, paths)

One might think the XSD already defines all elements and properties. So does Appendix B.
But what this this document does is to make the distinction between properties that are used as basic values or as symbolic references. And I claim this is necessary for a full semantic model of EDM.